client/src/components/Ticketing/ticketingmanager/AdminPurchase/utils/adminCommon.ts

```diff
@@ -0,0 +1,40 @@
+export type EventRow = {
+  id?: number;
+  desc: string;
+  eventid?: number;
+  eventinstanceid?: number;
+  eventname?: string;
+  eventdate?: string;
+  eventtime?: string;
+  ticketTypes?: string;
+  ticketRestrictionInfo?: ticketTypeRestriction[];
+  price?: number;
+  complimentary?: boolean;
+  availableseats?: number;
+  seatsForType?: number;
+  imageurl?: string;
+  qty?: number;
+  typeID?: number;
+};
+
+export interface ticketTypeRestriction {
+  id: number;
+  eventinstanceid: number;
+  tickettypeid: number;
+  concessionprice: number;
+  price: string;
+  ticketlimit: number;
+  ticketssold: number;
+  tickettypedescription?: string;
```

Comment:
I don't think this needs to be nullable as the ticket restriction API always returns a `description`